docs/migration-guide/to-17.md

```diff
@@ -132,6 +133,41 @@ root.walkRules((ruleNode) => {
 });
 ```
 
+### Changed `*-list` rules to have consistent behaviour for vendor prefixes and capitalisation
+
+We've changed the following `*-list` rules so that their treatment of vendor prefixes and capitalisation is consistent with the other `*-list` rules and with the `ignore*: []` secondary options:
```

Comment:
```suggestion
We've changed the following `*-list` rules so that their treatment of vendor prefixes and case is consistent with the other `*-list` rules and with the `ignore*: []` secondary options:
```